integration/__specs__/tests/analytics/deleteItems.spec.ts

```diff
@@ -0,0 +1,107 @@
+import { Mock } from '@Core/mock';
+import { CartPage } from '@Components/cartPage/cartPage/cartPage';
+import { GetCartItemsMock } from '@Mocks/api/mockio/v1/id/get';
+import { faker } from '@faker-js/faker';

```

Comment:
Where do you get faker from?

Reply:
but I don't see it in package.json

Reply:
maybe this is because flag --save-dev

Reply:
It is better to install it also in the project